packages/design-system/src/components/web-components/ds-modal-dialog/ds-modal-dialog.stories.tsx

```diff
@@ -0,0 +1,232 @@
+import { action } from '@storybook/addon-actions';
+import type { Meta } from '@storybook/react';
+import { useEffect } from 'react';
+import WebComponentDocTemplate from '../../../../../../.storybook/docs/WebComponentDocTemplate.mdx';
+import { webComponentDecorator } from '../storybook';
+import './ds-modal-dialog';
+import '../ds-button';
+
+const meta: Meta = {
+  title: 'Web Components/ds-modal-dialog',
+  args: {
+    alert: 'false',
+    'is-open': 'false',
+    children: (
+      <div>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed accumsan diam vitae metus
+          lacinia, eget tempor purus placerat.
+        </p>
+        <div slot="heading">
+          <h2>This is the modal heading.</h2>
+        </div>
+        <div slot="actions">
+          <form method="dialog">
+            <ds-button value="rstBtn">Reset</ds-button>
+            <ds-button type="submit" value="sbmBtn">
+              Confirm
+            </ds-button>
+          </form>
```

Comment:
Would it be a good idea to put some kind of long-form explanation on the doc page explaining how to set this up?